addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -845,6 +845,39 @@ def test_recompute_cash_rounding_lines(self):
         for move in moves_rounding:
             self.assertEqual(sum(move.line_ids.filtered(lambda line: line.display_type == 'rounding').mapped('balance')), moves_rounding[move])
 
+    def test_recompute_cash_rounding_lines_multi_company(self):
+        """
+        Ensure that when a move is created with cash rounding that will add an invoice line, the cash rounding accounts
+        will be that of the move's company and not the user's company.
+        """
+        cash_rounding_add_invoice_line = self.env['account.cash.rounding'].with_company(
+            self.company_data["company"]).create({
```

Comment:
Small indentation issue here
```suggestion
        cash_rounding_add_invoice_line = self.env['account.cash.rounding'].with_company(self.company_data["company"]).create({
```